src/lib/libraries/costumes.json

```diff
@@ -780,6 +780,17 @@
             1
         ]
     },
+    {
+        "name": "Empty",
+        "md5": "d36f6603ec293d2c2198d3ea05109fe0.png",
```

Comment:
Sorry, I made it a png by mistake. Will fix.

Reply:
Oh. I'll fix it. No worries.